File: Node.java
public class Node{

    private int load;
    private Node next;

    public Node (int load){
        this.load = load;
        this.next = null;
    }

    public int getLoad(){
        return load;
    }

    public void setLoad(int load){
        this.load = load;
    }

    public Node getNext(){
        return next;
    }

    public void setNext(Node next){
        this.next = next;
    }


}

File: Node.cpp
#include "Node.h"
Node::Node(){
	load = 0;
	next = NULL;
}

Node::Node(int n){
	load = n;
	next = NULL;
}

int Node::getLoad(){
	return load;
}

Node* Node::getNext(){
	return next;
}

void Node::setLoad(int n){
	load = n;
}

void Node::setNext(Node* n){
	next = n;
}
